sled-agent/src/services.rs

```diff
@@ -1931,43 +1948,33 @@ impl ServiceManager {
 
         // We initialize all the zones we can, but only return one error, if
```

Comment:
Can update the comment now that all errors get returned.

Reply:
Thanks!